examples/planning/add_keyword_plan.rb

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This example creates a keyword plan, which can be reused for retrieving
+# forecase metrics and historic metrics.
+
+require 'optparse'
+require 'google/ads/google_ads'
+require 'date'
+
+def add_keyword_plan(customer_id)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  keyword_plan = create_keyword_plan(client, customer_id)
+  plan_campaign = create_keyword_plan_campaign(
+    client,
+    customer_id,
+    keyword_plan
+  )
+  plan_ad_group = create_keyword_plan_ad_group(
+    client,
+    customer_id,
+    plan_campaign
+  )
+  create_keyword_plan_keywords(client, customer_id, plan_ad_group)
+  create_keyword_plan_negative_keywords(client, customer_id, plan_campaign)
+end
+
+def create_keyword_plan(client, customer_id)
+  keyword_plan = client.resource(:KeywordPlan)
+  keyword_plan.name = client.wrapper.string(
+    "Keyword plan for traffic estimate ##{(Time.new.to_f * 1000).to_i}"
+  )
+  forecast_period = client.resource(:KeywordPlanForecastPeriod)
+  forecast_period.date_interval =
+      client.enum(:KeywordPlanForecastInterval)::NEXT_QUARTER
```

Comment:
in all my examples I use the symbol literal e.g. `:NEXT_QUARTER`. Can we standardize on one style?

Reply:
Yes, now that I know this is possible it's clearly better.